back/engines/commercial/project_management/spec/models/email_campaigns/campaigns/moderator_digest_spec.rb

```diff
@@ -53,6 +53,13 @@
         command.dig(:event_payload, :top_ideas, 0, :author_name)
       ).to eq(expected_author_name)
     end
+
+    # added to reproduce the bug and test the fix
+    it 'gracefully handles absent projects' do
+      Project.find(moderator.roles[0]['project_id']).destroy!
+      commands = campaign.reload.generate_commands(recipient: moderator.reload).first
```

Comment:
FYI: I got interested in what these `commands` might be, and it turns out they may be redundant. See [slack thread](https://citizenlabco.slack.com/archives/C016C2EHURY/p1650638311690649). Not something for this PR, though.

Reply:
Really good point about the `commands` table. But still, it may be not easy to get rid of the `commands` array of hashes.

Reply:
Maybe you could add your concerns to the new [TEC ticket](https://citizenlab.atlassian.net/browse/TEC-174)? 🙏

Reply:
Done.